specs/latest/2.0/index.html

```diff
@@ -1952,11 +2068,14 @@ <h2 class="no-toc">Table of contents</h2>
 
     <h4><a name="readpixels">Reading back pixels</a></h4>
 
-    <p>Pixels in the current framebuffer can be read back into an ArrayBufferView object or a WebGLBuffer bound to the <code>PIXEL_PACK_BUFFER</code> target.</p>
+    <p>Pixels in the current framebuffer can be read back into an ByteBuffer object or a WebGLBuffer bound to the <code>PIXEL_PACK_BUFFER</code> target.</p>
```

Comment:
It's unfortunate to lose the ability to read back into ArrayBufferViews. Could this be changed to:

  ...read back into a ScalarDataSource object (ArrayBuffer, ArrayBufferView, or SharedArrayBuffer)

assuming SharedArrayBuffer support is added in this pull request. Otherwise: (ArrayBuffer or ArrayBufferView).